content/source/docs/cloud/workspaces/run-tasks.html.md

```diff
@@ -5,93 +5,91 @@ page_title: "Run Tasks - Workspaces - Terraform Cloud and Terraform Enterprise"
 
-The external system can then use this run information and respond back to Terraform Cloud with a passed or failed status. Terraform Cloud uses this status response to determine if a run should proceed, based on the task's enforcement settings within a workspace.
+The external service will use this run information to respond back to Terraform Cloud with a passed or failed status. Terraform Cloud uses this status response to determine if a run should proceed, based on the run task's enforcement settings that have been configured for the workspace.
 
 -> **API:** See the [Run Tasks APIs](../api/run-tasks.html).
 
 ## Requirements
 
-Run tasks can only be created on workspaces using a Terraform version v0.12+. Downgrading a workspace with existing run tasks to use a Terraform version v0.12 or prior will not result in an error, but the configured tasks will not execute.
+**Terraform Versions** - Run tasks can only be assigned to workspaces that use a Terraform version of 0.12 and later. Downgrading a workspace with existing run tasks to use a prior Terraform version will not result in an error, but the configured tasks will not execute as expected.
 
-You must be an organization owner to create a new task event hook. You must be at least a workspace administrator in order to connect a task event hook to a given workspace.
+**Permissions** - In order to create a run task, you must have a user account that has been assigned the organization owner permission. You must be at least a workspace administrator in order to associate a run task to a given workspace.
 
 ## Configuring a Run Task
 
-1. Navigate to the desired workspace, then select "Tasks" from the "Settings" menu. A page appears where you can either create a new event hook or select an existing one that is not connected to the workspace.
+1. Navigate to the desired workspace, then click **Run Tasks** from the **Settings** menu.
 
-![Screenshot: a workspace's settings drop-down menu](./images/run-tasks-workspace-settings.png)
+    ![Screenshot: a workspace's settings drop-down menu](./images/run-tasks-workspace-settings.png)
 
-2. Click "Create a new event hook." The "Task event hooks" page appears. 
+2. Click **Create a new run task** which will take you to the **Run Tasks** integration page.
 
-3. Click "Create hook."
+3. Click **Create run task**
 
-![Screenshot: an organization's blank task event hooks state](./images/run-tasks-blank-event-hooks.png)
+    ![Screenshot: an organization's blank run tasks state](./images/run-tasks-empty-run-tasks.png)
 
-4. Enter the information about the event hook to be configured:
+4. Enter the information about the run task to be configured:
```

Comment:
Just make sure that when you build locally this renders properly :-) Sometimes the renderer can be finicky about nesting bullets underneath numbered steps.